README.md

```diff
@@ -1,16 +1,73 @@
 # SmallCity
 
-| Created by Abakir Hanna, Ethan Mooney, and Lindsey Weiskopf
-
+| Created by Abakir Hanna, Ethan Mooney, and Lindsey Weiskopf [SmallCity Live Demo](https://step2020-smallcity.appspot.com/)
```

Comment:
You can move the live demo link to a second line still prefixed with "|"

Reply:
fixed